### ISSUE TITLE
How can I connect my Flutter app to two Firestore databases on two different Firebase Projects

### ISSUE THREAD

**Issue by OscarEReyes** [flutter/flutter#22195]
If I am unable to pass in my secondary FirebaseApp into FirebaseAuth is there another way to achieve this?
I can only access the second database if I leave it in test mode and allow third party read/writes.

**Comment by zoechi** [comment]
According to #13108 this should be supported already.

Can you please provide more information about how to reproduce? (code for minimal reproduction)

**Comment by zoechi** [comment]
Is this not a duplicate of #18279 ?

**Comment by OscarEReyes** [comment]
I just realized I did not read that issue correctly. Looking at it now, my reply to it makes no sense.

My apologies. Tried to get that working all day, and I guess I couldn't think straight while looking at similar issues/posting my own. Last night I thought it was a question regarding if it was possible to connect to multiple databases. 

Now well-rested, it is obvious I did not read the issue properly at all. I will close this issue and hope that one gets some sort of response in the future,
